src/modules/events/actions/log-handlers.js

```diff
@@ -112,7 +112,7 @@ export const handleInitialReportSubmittedLog = log => (dispatch, getState) => {
 export const handleMarketFinalizedLog = log => (dispatch, getState) => (
   dispatch(loadMarketsInfo([log.marketId], (err) => {
     if (err) return console.error(err)
-    const { volume, author, description } = getState().marketsData[log.marketId]
+    const { volume, author, description } = getState().marketsData[log.market]
```

Comment:
I see `log.marketId` still in use. Are both `log.market` and `log.marketId` defined?